public/components/data_connections/components/no_access.tsx

```diff
@@ -12,11 +12,11 @@ export const NoAccess = () => {
     <EuiPage>
       <EuiEmptyPrompt
         iconType="alert"
-        title={<h2>{'No permissions to access'}</h2>}
+        title={<h2>{'No permissions to access or resource does not exist'}</h2>}
```

Comment:
Fixed this. You are right in that we should be as specific as possible - they are most likely getting here via clicking a link in the table, but they can also manually change the link once they figure out the URL structure. Will keep it general for now, until UX comes back otherwise.